docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaEmailWebformHandler.php

```diff
@@ -221,24 +235,120 @@ protected function updateSubmissionWithFileAttachmentNames(array $fileAttachment
    *   Returns the submission contents as a human-readable list.
    */
   protected function formatSubmissionContentsAsList(array $submissionContents) {
-
-    $table = [
-      '#markup' => t('The following list contains the entire submission, and is formatted for ease of viewing and printing.'),
+    // Associative array of sections to groups of fields.
+    $keys_by_section = [
+      'Contact information' => [
+        'name_first',
+        'name_last',
+        'address_line1',
+        'address_line2',
+        'address_city',
+        'address_state_province',
+        'address_zip_postal_code',
+        'address_country',
+        'phone_number',
+        'fax_number',
+        'company_organization',
+        'email',
+      ],
+      'Request' => [
+        'request_id',
+        'confirmation_id',
+        'request_description',
+      ],
+      'Additional information' => [
```

Comment:
Let's called this "Supporting Documentation", since there could be another "Additional information" section at the bottom, if the webform has custom fields.